collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -505,4 +509,29 @@ public static Bitmap getBitmap(String path, BitmapFactory.Options originalOption
         Collections.addAll(mediaFiles, new File(mediaFilesDir).listFiles());
         return mediaFiles;
     }
+
+    public static void exportFormIndexToFile(FormIndex formIndex, File savepointIndexFile) {
+        try {
+            ObjectOutputStream oos = new ObjectOutputStream(new FileOutputStream(savepointIndexFile));
+            oos.writeObject(formIndex);
+            oos.flush();
+            oos.close();
+        } catch (Exception e) {
+            Timber.e(e);
+        }
+    }
+
+    public static FormIndex loadFormIndexFromFile() {
```

Comment:
If this doesn't require a `savepointIndexFile`, does the other one? I'd prefer to limit our `Collect.getInstance` but if we're only ever saving/loading the current index, shouldn't these calls mirror each other?

Reply:
I don't understand... it does require but this file is read a few lines below.

Reply:
Right but the signature of `exportFormIndexToFile` takes the savepointIndexfile, while this one retrieves it from the FormController. Shouldn't they mirror eachother? Is there a use case for specifying a different savepointIndexFile on export?